crypto/dh/dh_ameth.c

```diff
@@ -635,6 +635,8 @@ static int dh_cms_set_peerkey(EVP_PKEY_CTX *pctx,
         goto err;
     /* Get parameters from parent key */
     dhpeer = DHparams_dup(pk->pkey.dh);
+    if (!dhpeer)
```

Comment:
Please use `dhpeer == NULL`

Reply:
the same style if condition is a few lines above 
```C
    if (!pk)
        goto err;
```
and as well a few lines below (p is a pointer)
```C
    if (!p || !plen)
       goto err;
```
change these as well?

Reply:
It is up to you, if you want to. We should not introduce new `!pointer` conditions but it is absolutely not required to fix the whole file or function.